forms/viewsets/form.py

```diff
@@ -39,6 +43,11 @@ def get_queryset(self):
         )
         return queryset
 
+    def get_serializer_class(self):
+        if self.action in ("create", "metadata", "update", "partial_update"):
```

Comment:
There can't be "create" since FormViewSet does not have CreateModelMixin and therefore there is no create -function.

Reply:
very good point